aiounifi/__main__.py

```diff
@@ -51,10 +51,9 @@ def signalling_callback(signal, data):
 async def main(host, username, password, port, site, sslcontext=False):
     """Main function."""
     LOGGER.info("Starting aioUniFi")
-    loop = asyncio.get_event_loop()
```

Comment:
This is done inside the `asyncio.run()` method already, so it is redundant here. (In fact, it raises exceptions here for me on Python 3.7.7, so I'm not even sure how you were running this before...)

Reply:
I think Im running 3.7.6, no exceptions here :)